Header header
geometry_msgs/Pose pose
geometry_msgs/Twist  vel
float32 fuel
bool needTeleport